examples/stackdriver-trace/README.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+This example shows how to use [@opentelemetry/tracing](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-tracing) to instrument a simple Node.js application - e.g. a batch job.
+It supports exporting spans either to [Zipkin](https://zipkin.io) or to [Jaeger](https://www.jaegertracing.io).
```

Comment:
We should include Stackdriver here, as this example is targeted to Stackdriver exporter.